open-metadata-implementation/access-services/digital-service/digital-service-api/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/properties/ValidValuesMappingProperties.java

```diff
@@ -0,0 +1,199 @@
+/* SPDX-License-Identifier: Apache 2.0 */
```

Comment:
Confused about this?

Reply:
Absolutely , I haven't implemented the client yet and left these properties in until I was sure I wouldn't need them. They are just acting like a //TODO in this case.